docker/docker-compose.yml

```diff
@@ -60,12 +60,7 @@ services:
             echo waiting for spline
             sleep 5
           done
-          mvn test -P examples \
-            -D spline.producer.url=$${SPLINE_PRODUCER_URL} \
-            -D spline.mode=$${SPLINE_MODE} \
-            -D http.proxyHost=$${HTTP_PROXY_HOST} \
-            -D http.proxyPort=$${HTTP_PROXY_PORT} \
-            -D http.nonProxyHosts=$${HTTP_NON_PROXY_HOSTS}
+          mvn test -Pexamples -Dspline.producer.url=$${SPLINE_PRODUCER_URL} -Dspline.mode=$${SPLINE_MODE} -Dhttp.proxyHost=$${HTTP_PROXY_HOST} -Dhttp.proxyPort=$${HTTP_PROXY_PORT} -Dhttp.nonProxyHosts=$${HTTP_NON_PROXY_HOSTS}
```

Comment:
tried... no help :(

Reply:
I think I figured it out `>` is working, but when line is `more indented` the indentation is bigger then rest of the lines in that block, it will have newline included no matter what.

https://yaml.org/spec/1.1/#more%20indented%20line/

so the workaround could be:
 - keep indentation constant
 - use `"` code `"` and define new lines manually (using `\n`)

not sure if it's better at the end though.